dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/UserCommandKey.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.dynamometer.workloadgenerator.audit;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.io.WritableComparable;
+
+
+public class UserCommandKey implements WritableComparable {
+  private Text user;
+  private Text command;
+
+  public UserCommandKey() {
+    user = new Text();
+    command = new Text();
+  }
+
+  public UserCommandKey(Text user, Text command) {
+    this.user = user;
+    this.command = command;
+  }
+
+  public UserCommandKey(String user, String command) {
+    this.user = new Text(user);
+    this.command = new Text(command);
+  }
+
+  public String getUser() {
+    return user.toString();
+  }
+
+  public void setUser(String user) {
```

Comment:
Are these setters used for anything? Does the framework require them?

Reply:
Or the getters, for that matter?

Reply:
The getters are necessary for the equality check; the setters can probably be omitted.

Reply:
sgtm